### PR TITLE
docs: fix access control migration link

### PR DESCRIPTION
**Pull request by emonddr**
Signed-off-by: Dominique Emond <dremond@ca.ibm.com>

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by jannyHou — APPROVED** [review]
👍

File: docs/site/RBAC-with-authorization.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ The architecture diagram is:
 The example is created in repository
 [access-control-migration](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration),
 and a tutorial of building it from scratch can be found in
-[access control example migration](./migration/auth/example.md).
+[access control example migration](https://loopback.io/doc/en/lb4/migration-auth-access-control-example.html).
```
Comment (dhmlau) [inline docs/site/RBAC-with-authorization.md:50]: with the convention that the `.md` file name matching the url, I wonder if it's better to rename `example.md` to `migration-auth-access-control-example.md`.

Comment (nabdelgadir) [inline docs/site/RBAC-with-authorization.md:50]: With the rename, does the relative path work?

**Review by jannyHou — APPROVED** [review]
I am glad you noticed there are more than one place needs update. Still I am not sure why the old relative path was not rendered, the new one looks good to me and hope it works.

File: docs/site/RBAC-with-authorization.md (hunk 1 above)
Comment (emonddr) [reply docs/site/RBAC-with-authorization.md]: > With the rename, does the relative path work?

I agree, let's try `migration-auth-access-control-example.md` first.

**Review by nabdelgadir — APPROVED** [review]

**Comment by dhmlau** [comment]
Verified with my local preview. 👍

**Pull request merged**